server/src/controllers/seriesController.ts

```diff
@@ -22,6 +22,22 @@ class SeriesController {
       return next(ApiError.internal("Something went wrong, please try again"));
     }
   }
+  async mainSeries(req: Request, res: Response, next: NextFunction) {
+    try {
+      const allSeries = await Catalog.findAll();
+      const latestSeries = allSeries.slice(-10);
+      const topTen: number[] = [21, 18, 11, 42, 22, 47, 41, 31, 15, 5];
```

Comment:
Please add a comment that these data are fake and will be replaced with real logic for sorting top ten series